shared/router-v2/tab-bar/container.desktop.tsx

```diff
@@ -1,29 +1,35 @@
 import * as Tabs from '../../constants/tabs'
 import * as Chat2Gen from '../../actions/chat2-gen'
 import * as ConfigGen from '../../actions/config-gen'
+import * as LoginGen from '../../actions/login-gen'
 import * as ProfileGen from '../../actions/profile-gen'
 import * as PeopleGen from '../../actions/people-gen'
 import * as RouteTreeGen from '../../actions/route-tree-gen'
 import * as SettingsConstants from '../../constants/settings'
 import * as TrackerConstants from '../../constants/tracker2'
-import TabBar from './index.desktop'
-import {connect} from '../../util/container'
+import TabBar, {Props} from './index.desktop'
+import {connect, TypedState} from '../../util/container'
 import {memoize} from '../../util/memoize'
 import {isLinux} from '../../constants/platform'
 import openURL from '../../util/open-url'
 import {quit, hideWindow} from '../../util/quit-helper'
 import {tabRoots} from '../routes'
 import * as RPCTypes from '../../constants/types/rpc-gen'
 import * as SettingsGen from '../../actions/settings-gen'
+import * as ProvisionGen from '../../actions/provision-gen'
+import * as SignupGen from '../../actions/signup-gen'
 
 type OwnProps = {
   navigation: any
   selectedTab: Tabs.AppTab
 }
 
-const mapStateToProps = state => ({
+const mapStateToProps = (state: TypedState) => ({
   _badgeNumbers: state.notifications.navBadges,
   _walletsAcceptedDisclaimer: state.wallets.acceptedDisclaimer,
+  accountRows: state.config.configuredAccounts
```

Comment:
the fact that this is done at least 3x implies we should make this a utility in constants i think